core/trino-main/src/main/java/io/trino/sql/planner/plan/TableWriterNode.java

```diff
@@ -243,7 +240,13 @@ public ConnectorTableMetadata getTableMetadata()
         @Override
         public String toString()
         {
-            return catalog + "." + tableMetadata.getTable();
+            return getCatalogName() + "." + tableMetadata.getTable();
```

Comment:
nit: Usually fields are used directly in the `toString` implementation

Reply:
You right, will be changed.